### PR TITLE
feat: add soft scan trigger

### PR DESCRIPTION
**Pull request by flashmatt**
Adding Soft Scan Trigger to enable software scanning in addition to the physical trigger. Tested on a TC26 & TC21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ npx cap sync
 * [`disable()`](#disable)
 * [`enableScanner()`](#enablescanner)
 * [`disableScanner()`](#disablescanner)
+* [`startScanning()`](#startscanning)
+* [`disableScanner()`](#stopscanning)
```
Comment (jkbz64) [inline README.md]: Probably forgot to docgen regenerate before pushing? should be stopScanning() here

File: android/src/main/java/com/jkbz/capacitor/datawedge/DataWedge.java [hunk 2]
```diff
@@ -38,5 +38,21 @@ public Intent disableScanner() {
...
+        return intent;
+    }
+
+    public Intent stopScanning() {
+        Intent intent = new Intent();
+        intent.setAction(DATAWEDGE_PACKAGE + ".ACTION");
+        intent.putExtra("com.symbol.datawedge.api.SOFT_SCAN_TRIGGER", "STOP_SCANNING");
+
+        return intent;
+        }
```
Comment (jkbz64) [inline android/src/main/java/com/jkbz/capacitor/datawedge/DataWedge.java]: indentation doesn't seem right

File: src/definitions.ts [hunk 3]
```diff
@@ -49,12 +49,30 @@ export interface DataWedgePlugin {
...
+   */
+  stopScanning(): Promise<void>;
+
   /**
   * Listen for successful barcode readings
   * 
   * ***Notice:*** Requires intent action to be set to `com.capacitor.datawedge.RESULT_ACTION` in current DataWedge profile (it may change in the future)
   *
-  * @since 0.1.0
+  * @since 0.1.2
```
Comment (jkbz64) [inline src/definitions.ts]: this change doesn't seem related to the PR

**Review by jkbz64 — CHANGES_REQUESTED** [review]
Some small mistakes here and there but nevertheless thank you very much for contributing :+1:  
I will merge it once you regenerate docs and fix the small issues, or do you want me to do it?
Also would appreciate if the commit followed angular commit convention/conventional commit so it would be named something like `feat: add soft scan trigger`

**Comment by flashmatt** [comment]
No problem, thanks for writing the initial code! I'll make those changes. Sorry for the mistakes its my first pull

**Comment by jkbz64** [comment]
Thanks :+1:
